velox/docs/functions/spark/conversion.rst

```diff
@@ -2,6 +2,29 @@
 Conversion Functions
 ====================
 
+Conversion Functions
+--------------------
+
+.. spark:function:: cast(value AS type) -> type
+
+    Explicitly cast a value as a type. As ANSI mode is OFF, an overflow will not cause an error but 
+    instead will “wrap” the result. The ``value`` with an invalid format or invalid characters for 
+    ``type`` will result in a NULL. ::
+
+      SELECT cast(128 as tinyint); -- -128
+      SELECT cast('2012-Oct-23' as date); -- NULL
+
+.. spark:function:: try_cast(value AS type) -> type
+
+    Returns the ``value`` cast to data type ``type`` if possible, or NULL if not possible.
+    try_cast differs from cast function in following cases: If the ``value`` cannot fit within 
```

Comment:
try_cast differs from cast function in following cases: 

- If the ``value`` cannot fit within